src/components/organisms/PageHeader/PageHeader.styled.tsx

```diff
@@ -114,3 +119,50 @@ export const Divider = styled(RawDivider)`
   margin-right: 1rem;
   top: 0;
 `;
+
+export const ActiveProjectButton = styled(RawButton).attrs({
+  type: 'text',
+  ghost: true,
+})`
+  display: flex;
+  align-items: center;
+  margin-left: 16px;
+  padding: 0px;
+  width: 100%;
+  background-color: transparent !important;
+  :hover {
+    background-color: transparent;
+    & span {
+      color: ${Colors.geekblue9} !important;
+    }
+  }
+`;
+
+export const ProjectName = styled.span`
+  width: 100%;
+  max-width: 272px;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+  color: ${Colors.grey9};
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 20px;
+  text-align: left;
+  transition: all 0.5s ease-in;
+
+  &:disabled {
+    color: inherit !important;
+  }
+`;
+
+export const MenuOutlinedIcon = styled(RawMenuOutlined)`
+  color: ${Colors.grey9};
+  margin-right: 8px;
+
+  svg {
+    width: 20px;
```

Comment:
Do we need to set a width and height? Can't we use directly the font-size property on the `MenuOutlinedIcon` component?